### PR TITLE
Fix test failure in DefaultRequestTargetTest

### PR DESCRIPTION
**Pull request by minwoox**

### CODE REVIEW

File: core/src/test/java/com/linecorp/armeria/internal/common/DefaultRequestTargetTest.java [hunk 1]
```diff
@@ -347,9 +347,9 @@ void shouldNormalizeAmpersand(Mode mode) {
     @Test
     void serverShouldRemoveMatrixVariablesWhenNotAllowed() {
         // Not allowed
-        assertAccepted(forServer("/;a=b?c=d;e=f"), "/", "c=d;e=f");
```
Comment (jrhee17) [inline core/src/test/java/com/linecorp/armeria/internal/common/DefaultRequestTargetTest.java:350]: Actually, why is this failing?

Comment (minwoox) [reply core/src/test/java/com/linecorp/armeria/internal/common/DefaultRequestTargetTest.java:350]: Matrix variables are followed by a path variable so it shouldn't be right after the slash. Isn't it right?

Comment (jrhee17) [reply core/src/test/java/com/linecorp/armeria/internal/common/DefaultRequestTargetTest.java:350]: Sorry, I left a comment on the wrong line 😅    I actually meant L352.

Comment (minwoox) [reply core/src/test/java/com/linecorp/armeria/internal/common/DefaultRequestTargetTest.java:350]: It isn't failed. 😅 Let me revert the change. 😉

**Review by ikhoon — APPROVED** [review]
Thanks, @minwoox! 👍

**Review by jrhee17 — APPROVED** [review]
Thanks for the quick fix 👍 👍 👍

**Comment by minwoox** [comment]
Let me merge this for CI to pass.

**Pull request merged**